### PR TITLE
Added 1,500+ words

### PR DESCRIPTION
**Pull request by N8-Des**
This was a great use of a Friday

### CODE REVIEW

**Comment by JCharante** [comment]
Please don't group the words under the array "Nate", but under the Array with 1000 words it in already

**Comment by N8-Des** [comment]
okay I will do that, I didn't know if it was okay because of the lack of alphabetical order.

**Comment by JCharante** [comment]
It's okay, I don't really care about that.

**Comment by JCharante** [comment]
Please change the word's locations from "More_Words" to "approved_words"

**Comment by JCharante** [comment]
Have you tested it w/ example.py? I don't think it would run as you defined "approved_words" twice.

File: Smelter/word_whitelist.json [hunk 1]
```diff
@@ -1,1187 +1 @@
...
..."to be searched, and the persons or things to be seized. AMENDMENT V Share No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation. AMENDMENT VI Share In all criminal prosecutions, the accused shall enjoy the right to a speedy and public trial, by an impartial jury of the State and district wherein the crime shall have been committed, which district shall have been previously ascertained by law, and to be informed of the nature and cause of the accusation; to be confronted with the witnesses against him; to have compulsory process for obtaining witnesses in his favor, and to have the Assistance of Counsel for his defence. AMENDMENT VII Share In Suits at commo
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:1133]: You just deleted everything in the file?

**Comment by N8-Des** [comment]
yup. You don't get to steal my work after kicking me.

**Comment by JCharante** [comment]
You know that's not how git works, right? lol

**Pull request closed without merging**